app/assets/stylesheets/_tab-navigation.scss

```diff
@@ -37,25 +37,25 @@
 }
 
 .app-tab-navigation__item--current {
-  @include govuk-media-query($until: tablet) {
-    border-left: 4px solid $govuk-link-colour;
+  @include all.govuk-media-query($until: tablet) {
+    border-left: 4px solid govuk-link-colour;
     padding-left: 11px;
   }
 
-  @include govuk-media-query($from: tablet) {
-    border-bottom: 4px solid $govuk-link-colour;
+  @include all.govuk-media-query($from: tablet) {
+    border-bottom: 4px solid govuk-link-colour;
     padding-left: govuk-spacing(3);
   }
 }
 
 .app-tab-navigation__link {
-  @include govuk-link-common;
-  @include govuk-link-style-no-visited-state;
-  @include govuk-link-style-no-underline;
-  @include govuk-typography-weight-bold;
+  @include all.govuk-link-common;
+  @include all.govuk-link-style-no-visited-state;
+  @include all.govuk-link-style-no-underline;
+  @include all.govuk-typography-weight-bold;
 
   &:not(:focus):hover {
-    color: $govuk-link-colour;
+    color: govuk-link-colour;
```

Comment:
Again, this seems incorrect. We may end up using other components in the GOV.UK ecosystem, which would rely on these variables.